corehq/apps/cloudcare/static/cloudcare/js/formplayer/utils/util.js

```diff
@@ -128,6 +128,23 @@ hqDefine("cloudcare/js/formplayer/utils/util", function () {
         };
     };
 
+    Util.getStickyQueryInputs = function () {
+        if (!hqImport("hqwebapp/js/toggles").toggleEnabled('WEBAPPS_STICKY_SEARCH')) {
+            return {};
+        }
+        if (!this.stickyQueryInputs) {
+            return {};
+        }
+        return this.stickyQueryInputs[sessionStorage.queryKey] || {};
+    };
+
+    Util.setStickyQueryInputs = function (inputs) {
+        if (!this.stickyQueryInputs) {
```

Comment:
Should this create the object if it doesn't exist but valid inputs are submitted? I am not totally sure, where it is initialized

Reply:
Not quite following - this is where it's initialized.

Reply:
ugh i very much misread this and added an `else` around line 143 that does not exist.